tools/api-markdown-documenter/src/RenderMarkdown.ts

```diff
@@ -28,22 +28,6 @@ export interface RenderApiModelAsMarkdownOptions
 /**
  * Renders the provided model and its contents, and writes each document to a file on disk.
  *
- * @remarks
```

Comment:
All of this context is covered by property docs in `RenderDocumentsAsMarkdownOptions`.